src/main/java/uk/gov/justice/hmpps/prison/service/transformers/OffenderChargeTransformer.kt

```diff
@@ -27,6 +28,8 @@ class OffenderChargeTransformer : Converter<OffenderCharge, OffenceHistoryDetail
       .secondaryResultConviction(offenderCharge.resultCodeTwo?.isConvictionFlag)
       .courtDate(latestCourtEvent?.eventDate)
       .offenceSeverityRanking(offenderCharge.offence.severityRanking?.toInt() ?: Int.MAX_VALUE)
+      .sentenceStartDate(sentence?.sentenceStartDate)
+      .primarySentence(if (sentence != null) sentence.consecutiveToSentenceSequence == null else null)
```

Comment:
only set to `true` or `false` if there is a sentence

Reply:
I'd normally use `?` for this, e.g. `sentence?.let { it.consecutuveToSequence == null }`